packages/godwoken/src/types.ts

```diff
@@ -38,7 +38,7 @@ export interface L2Transaction {
 }
 
 export interface L2TransactionWithStatus {
-  transaction: L2Transaction;
+  transaction: L2Transaction | undefined;
```

Comment:
https://github.com/nervosnetwork/godwoken/blob/c4be58f30744aef83717e2a12d60fe4d50b165ab/crates/jsonrpc-types/src/godwoken.rs#L522
It's a bug before, godwoken's RPC response `L2TransactionWithStatus.transaction` is an option.
When transaction not found, it may return `{tx: null, status: pending}`

Reply:
Oh sorry
https://github.com/nervosnetwork/godwoken/blob/5d11b3b2c30133e0b3dc2e9553e6bd04f5e4dbb7/docs/RPC.md#method-gw_get_transaction
Some misunderstandings before.
Only when `verbose = 1` will return `{tx: null, status: xxx}`
And this change reverted